lib/message/utils.js

```diff
@@ -107,22 +106,24 @@ export async function handleVerificationResult({ data, filename = 'msg.txt', sig
         });
         const verificationResults = await Promise.all(verificationPromises);
 
-        return verificationResults.filter(({ verified }) => verified === SIGNED_AND_VALID).reduceRight(
-            (acc, result) => {
-                if (acc.verified !== SIGNED_AND_VALID) {
-                    acc.verified = result.verified;
-                    acc.data = arrayToBinaryString(result.data);
+        return verificationResults
+            .filter(({ verified }) => verified === SIGNED_AND_VALID)
+            .reduceRight(
+                (acc, result) => {
+                    if (acc.verified !== SIGNED_AND_VALID) {
```

Comment:
Is it really possible to have `acc.verified !== SIGNED_AND_VALID` if we just filtered `verificationResults` like that `filter(({ verified }) => verified === SIGNED_AND_VALID)`?

Reply:
Yes you are right